workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -154,4 +154,47 @@
       expect(output_string).to eq(expected_output)
     end
   end
+
+  describe '#fetch_queue_build' do
+    queue_build_json_str = '{"count": 2,
+                             "build": [{"buildTypeId": "CozmoOne_MasterBuild",
+                                        "branchName": "BI-123-XYZ",
+                                        "id": "4307"},
+                                       {"buildTypeId": "OverDriveOne_Release_Build",
+                                        "branchName": "BI-124-XYZ",
+                                        "id": "4308"}]}'
+    before do
+      stub_request(:get, /buildQueue/).to_return(:status => 200, :body => queue_build_json_str)
+    end
+    it 'buildTypeId is found that returns with the queue build correctly' do
+      expected_output = "\n<#{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=4307"\
+                        "|CozmoOne_MasterBuild> - BI-123-XYZ"
+      output_string = subject.fetch_queue_build("CozmoOne_MasterBuild")
+      expect(output_string).to eq(expected_output)
+     end
+
+    it 'build_id_wildcard == nill that returns with list of all found queue builds' do
+      expected_output = "\n<#{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=4307"\
+                        "|CozmoOne_MasterBuild> - BI-123-XYZ"\
+                        "\n<#{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=4308"\
+                        "|OverDriveOne_Release_Build> - BI-124-XYZ"
+      output_string = subject.fetch_queue_build(nil)
+      expect(output_string).to eq(expected_output)
+    end
+
+    it "buildTypeId isn't found that returns empty"  do
```

Comment:
this description is not correct, also it doesn't cover enough. It should be `returns only build that matches build_id_wildcard`  when you input `build_id_wildcard` that only matches with one of two builds.